app/lib/company_pages_analytics.rb

```diff
@@ -1,8 +1,9 @@
 class CompanyPagesAnalytics
+  MANUALLY_ADDED_PAGES_TYPES = [Page::CHROME_EXTENSION, Page::BY_AJAX].freeze
   def self.count_pages(current_tenant)
     current_tenant.pages.where(
-      'page_type != ? and pages.created_at > ?',
-      Page::CHROME_EXTENSION, 1.day.ago
+      'page_type not in (?) and pages.created_at > ?',
```

Comment:
NOT IN is used for arrays
find another SQL operator for "!="

Reply:
but I have array here
![image](https://user-images.githubusercontent.com/27546645/35813782-27378260-0a9d-11e8-8886-06804b0e22f8.png)

Reply:
yes. now yes

Reply:
so, it's good or I must change it?